manifests/container-storage-interface/oci-csi-controller-driver.yaml

```diff
@@ -74,6 +90,23 @@ spec:
               readOnly: true
             - mountPath: /var/run/shared-tmpfs
               name: shared-tmpfs
+        - name: oci-csi-fss-controller-driver
+          args:
+            - --v=2
+            - --endpoint=unix://var/run/shared-tmpfs/csi-fss.sock
+          command:
+            - /usr/local/bin/oci-csi-fss-controller-driver
+          image: iad.ocir.io/oracle/cloud-provider-oci:latest
```

Comment:
This should be the exact version corresponding to this release

Reply:
Changed in a later commit.